doc/Makefile

```diff
@@ -29,18 +29,26 @@ ESSINFONODE1='* ESS: (ess).'
 ESSINFONODE2='	        Emacs Speaks Statistics'
 ESSINFONODE3='                        (S/S+/R, SAS, BUGS, Stata, XLisp-Stat).'
 
-### Targets --
 PDFs = ess.pdf readme.pdf refcard/refcard.pdf
 TXTs = ../README ../ANNOUNCE ../NEWS ../ONEWS
 
-all  : info text html pdf
-docs : info text html
+### Targets --
+ALL_TARGETS = info text
```

Comment:
I think those BUILD_XYZ_DOCS config are not particularly useful? IIt would be simpler to have those html and pdf targets run on travis and have them in the tarball dependencies.

Reply:
OK, done